### PR TITLE
Add a first version of the DB import pipeline

### PR DESCRIPTION
**Pull request by beckerhe**
This is just a first version. Some tests and some documentation is still missing. But since it's already running, I thought it's better to submitt it here as well.

### CODE REVIEW

File: tools/db_import/config.yml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    cloud_function_name: oobi-postsubmit-importer-v7
+    table_name: benchmarking_experiments.oobi_v7
+    sql_delete: *common_sql_delete
+    sql_data_present: *common_sql_data_present
+    service_account: oobi-db-import
+    rules:
+    - filepath_regex: "^(?P<fullpath>(?P<prefix>(?P<github_action_run_id>[0-9]+)/(?P<github_action_run_attempt>[0-9]+))/benchmark-results/benchmark-results-(?P<config>.*)\\.json)$"
+      sql_condition: *common_sql_condition
+      result: |
+        // Import helper functions
```
Comment (pzread) [inline tools/db_import/config.yml:62]: This seems to be a big chunk of code. Is it possible to put them in a file (and we can get proper syntax highlights)

**Comment by pzread** [comment]
Is it possible to submit these code in multiple PRs with each PR focusing on one subject? That might make review easier : )

File: tools/db_import/config.yml (hunk 1 above)
Comment (beckerhe) [reply tools/db_import/config.yml:62]: Probably. Let me explore this option!

**Comment by beckerhe** [comment]
> Is it possible to submit these code in multiple PRs with each PR focusing on one subject? That might make review easier : )

Yes sure. Let me split this up. (This will take a while though).

**Comment by pzread** [comment]
> > Is it possible to submit these code in multiple PRs with each PR focusing on one subject? That might make review easier : )
> 
> Yes sure. Let me split this up. (This will take a while though).

Thanks. I can still read this PR to understand some details, which hopefully will speed up the reviews on splitted PRs

**Pull request closed without merging**